toponetx/classes/cell_complex.py

```diff
@@ -2234,10 +2234,8 @@ def get_linegraph(self, s=1, cells=True):
         nx.Graph
             A NetworkX graph representing the s-linegraph of the Cell Complex.
         """
-        if isinstance(s, None):
-            ValueError(
-                "s must be a positive integer larger than 1, got type of s None."
-            )
+        if not isinstance(s, int):
+            ValueError("s must be a positive integer larger than 1.")
```

Comment:
The check and error message do not align. It is not checked whether `s` is larger than 1, just that it is an int.

Reply:
fixed